pkgs/development/python-modules/django-celery-beat/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, python-crontab
+, celery
+, django-timezone-field
+, tzdata }:
+
+buildPythonPackage rec {
+  pname = "django-celery-beat";
+  version = "2.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-uiT4btlWug7itDI3pJMD6/Wqfg+wzLfgVCt+MaRj3Lo=";
+  };
+
+  propagatedBuildInputs = [
+    python-crontab
+    celery
+    django-timezone-field
+    tzdata
+  ];
+
+  # Tests stuck at test_model.py
+  doCheck = false;
```

Comment:
Then we should disable that one check or file instead of everything.

Reply:
Tried it but somehow I'm unable to disable specific tests. Nether by name or file path :(